src/products/components/ProductListPage/filters.ts

```diff
@@ -76,15 +78,26 @@ const messages = defineMessages({
   }
 });
 
+const extractAttributes = (opts: ProductListFilterOpts) => (
```

Comment:
Most if not whole project uses maps and filters. What do you think that instead of currying we'd implement something like this?

```JSX
function filterByType(type: AttributeInputTypeEnum) {
  return (attribute: attributetype) => attribute.inputType === type
}

const attributes = opts.attributes;
const booleanAttributes = attributes.filter(filterByType(AttributeInputTypeEnum.BOOLEAN))
...
```

Reply:
Ur playground so what ever is more convenient for u 👌